compiler/src/openxla/compiler/nvgpu/Transforms/BUILD.bazel

```diff
@@ -34,8 +34,10 @@ cc_library(
     deps = [
         ":PassesIncGen",
         "//compiler/src/openxla/compiler/nvgpu:defs",
+        "//compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/IR",
         "@llvm-project//mlir:IR",
         "@llvm-project//mlir:Pass",
+        "@mlir-hlo//:mlir_hlo",
```

Comment:
Let's perhaps flip this to StableHLO for now (mlir-hlo repo is due for changes and I don't think the difference is meaningful for what we need here).

Reply:
Done.